File: src/test/samples/invalidSamples/invalidSample.hs
-- #region FirstRegion
x = 42
-- #endregion

-- #endregion Invalid end boundary
-- #region Invalid start boundary

-- #region Second Region  
data MyClass = MyClass
    --    #region   InnerRegion  
  deriving Show
    --         #endregion    ends InnerRegion  

  --  #region  
deriving UnnamedRegion
--  #endregion  
-- #endregion

File: src/test/samples/invalidSamples/invalidSample.js
//#region FirstRegion
const x = 42;
//#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

// #region Second Region
class MyClass {
  //    #region   InnerRegion
  method() {}
  //      #endregion   ends InnerRegion

  // #region
  anotherMethod() {}
  //#endregion
}

//  #endregion
